lib/commander/runner.rb

```diff
@@ -332,19 +332,27 @@ def remove_global_options(options, args)
       options.each do |option|
         switches = option[:switches]
         next if switches.empty?
-        switch_has_arg = switches.any? { |s| s =~ /[ =]/ }
+        option_has_argument = switches.any? { |s| s =~ /[ =]/ }
```

Comment:
What do you think about changing the name to `option_takes_argument` — just to clarify that the option *may* have an argument, rather than suggesting something about the current state?

Reply:
👍  That's much clearer.